src/v2/Apps/Auction/Components/OnSubmitValidationError.tsx

```diff
@@ -18,15 +21,15 @@ export type TrackErrors = (errors: string[]) => void
     https://github.com/jaredpalmer/formik/issues/1484#issuecomment-490558973
  */
 
-export const OnSubmitValidationError: React.FC<{
-  cb: TrackErrors
-  formikProps: Partial<FormikProps<FormValuesForRegistration>>
-}> = props => {
-  const { cb, formikProps } = props
+export const OnSubmitValidationError: React.FC<{ cb: TrackErrors }> = ({
+  cb,
+}) => {
+  const formikProps = useFormikContext<
+    BillingInfoWithTerms | BillingInfoWithBid
+  >()
 
   const effect = () => {
```

Comment:
This seems to be [copied directly from a suggested temporary solution](https://github.com/jaredpalmer/formik/issues/1484#issuecomment-490558973) for a missing property on Formik to pass a callback for when the submission doesn't happen (in our case, for error tracking).

I rewrote the component, keeping the same dependencies. Adding `isValid` and `errors` makes the callback execute one extra time, and I don't think it's worthwhile spending time on fixing it since **the entire component must be removed when Formik implements the feature.**